parser/src/main/scala/it/pagopa/interop/commons/parser/InterfaceParserUtils.scala

```diff
@@ -21,11 +39,25 @@ object InterfaceParserUtils {
 
   implicit val openApiInterfaceExtractor: InterfaceParserUtils[Json] = new InterfaceParserUtils[Json] {
     override def getUrls(serviceInterface: Json): Either[Throwable, List[String]] =
-      serviceInterface.hcursor.downField("servers").as[List[Json]].flatMap(_.traverse(_.hcursor.get[String]("url")))
+      serviceInterface.getVersion.flatMap {
+        case `2.0`                                 => serviceInterface.hcursor.downField("host").as[String].map(List(_))
+        case `3.0.0` | `3.0.1` | `3.0.2` | `3.0.3` =>
+          serviceInterface.hcursor.downField("servers").as[List[Json]].flatMap(_.traverse(_.hcursor.get[String]("url")))
+        case `3.1.0`                               =>
+          serviceInterface.hcursor.downField("servers").as[List[Json]].flatMap(_.traverse(_.hcursor.get[String]("url")))
```

Comment:
Just minor semplification
```suggestion
        case `3.0.0` | `3.0.1` | `3.0.2` | `3.0.3` | `3.1.0` =>
          serviceInterface.hcursor.downField("servers").as[List[Json]].flatMap(_.traverse(_.hcursor.get[String]("url")))
```

Reply:
I do it explicitly to keep more evidence on the version (based on minor). 
@TonioGela proposed the same solution suggested here.
Nothing against it.
I'm going to change it.